### ISSUE TITLE
Display Load and Grid current on card like solar, battery and inverter

### ISSUE THREAD

**Issue by codifierr** [slipx06/sunsynk-power-flow-card#265]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Currently, the card doesn't show load and grid current
<img width="537" alt="Screenshot 2024-02-23 at 11 59 37 PM" src="https://github.com/slipx06/sunsynk-power-flow-card/assets/12495994/94d2c968-e7d3-49e0-83e3-fd35a9838cb6">

### Expected behaviour

The card should show the load and grid current.

### Mode

Both

### Context / Reason

The card shows Battery, solar, and Inverter. It can also show the Load and Grid current.

**Comment by slipx06** [comment]
Yes, the problem is there is not enough free space when some of the other features are enabled i.e. 3 phase, AUX, non-essential, so you have to design conditional checks to only display this data when some of those options are not enabled. Will keep it in mind for a future release

**Comment by codifierr** [comment]
Thanks for considering this request.

**Comment by Roving-Ronin** [comment]
@codifierr you could always use that box to display the grid Voltage, Frequency and Load (assume you mean the overall total kW being used in the house from all sources?). Just use the applicable sensors for those instead of the default ones.

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/108674933/9d22cc94-f871-4086-9533-2889c0135d7e)

This is what I do for the Huawei inverters, in the screenshot above, we can see:

242.6V Grid Voltage
50Hz Frequency
9.48kW Total House Load  (then looking at other boxes, quickly it can be seen that 9.38kW is coming from solar [or battery] and 93W is coming from the grid).
